robottelo/cli/content_credentials.py

```diff
@@ -0,0 +1,39 @@
+# -*- encoding: utf-8 -*-
+"""
+Usage::
+
+    hammer content-credentials [OPTIONS] SUBCOMMAND [ARG] ...
+
+Parameters::
+
+    SUBCOMMAND                    subcommand
+    [ARG] ...                     subcommand arguments
+
+Subcommands::
+
+    create                        Create a content credential
+    delete                        Destroy a content credential
+    info                          Show a content credential
+    list                          List content credentials
+    update                        Update a content credential
+"""
+from robottelo.cli.base import Base
+
+
+class ContentCredential(Base):
```

Comment:
That's ContentCredential or ContentCredentials (in plural)?

I believe usually we name class after hammer command, and they tend to be singular.

Reply:
Singular:

    `~]# hammer content-credential -h

    Usage:
        hammer [OPTIONS] SUBCOMMAND [ARG] ...`

Reply:
Its plural in the docblock on L5. Inconsequential, but inconsistent.

Reply:
sorry about that, fixing now. The hammer  command uses singular, so we need that for commands, but there will be one other content-credential, so not sure if module needs to be singular